lib/webauthn/authenticator_attestation_response.rb

```diff
@@ -98,5 +115,29 @@ def raw_subject_key_identifier(certificate)
       ext_value = ext_asn1.value.last
       OpenSSL::ASN1.decode(ext_value.value).value
     end
+
+    def attestation_root_certificates_store
+      certificates = nil
+      WebAuthn.configuration.attestation_root_certificates_finders.each do |finder|
+        certificates = finder.find(attestation_format,
+                                   aaguid: aaguid,
+                                   attestation_certificate_key_id: attestation_certificate_key)
+        break if certificates.any?
```

Comment:
Instead of `each` + `break`, I'd rewrite this using [`detect`](https://ruby-doc.org/core-2.6.5/Enumerable.html#method-i-detect) and have this line be `certificates if certificates.any?`